provider/kubernetes/kubernetes.go

```diff
@@ -315,6 +309,48 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 	return &templateObjects, nil
 }
 
+func getEntrypoints(i *v1beta1.Ingress) []string {
+	entrypointsAnnotation, ok := i.Annotations[types.LabelFrontendEntryPoints]
+	if ok {
+		return strings.Split(entrypointsAnnotation, ",")
+	}
+	return nil
+}
+
+func getRulePassHostHeader(i *v1beta1.Ingress, p *Provider) bool {
```

Comment:
It's not really a _rule_ it seems to me. Can we drop that part from the function name?

Reply:
Done.